LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ConcentratorDeduplication.cs

```diff
@@ -74,6 +87,7 @@ private void AddToCache(string key, StationEui stationEui)
                 SlidingExpiration = DefaultExpiration
             });
 
-        public void Dispose() => this.cache.Dispose();
+        public void Dispose()
+            => this.cache.Dispose();
```

Comment:
nit: Revert as this change seems unrelated?

```suggestion
        public void Dispose() => this.cache.Dispose();
```